recipes/iris-esmf-regrid/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "iris-esmf-regrid" %}
+{% set version = "0.1.dev0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SciTools-incubator/iris-esmf-regrid/archive/v{{ version }}.tar.gz
+  sha256: 0d8d4b4dd73f32dd5d8f4789838e0ca76919373f216115266274bc5481fcaf0f
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
@jamesp Sorry, forgot to mention that you should add in `--no-deps` here, as the recipe will have accounted for all the dependencies within `conda` 👍

Reply:
Done